### DISCUSSION TITLE
Filter out null FileType results

### DISCUSSION THREAD

**Discussion by JonoSuave** [microsoft-search/pnp-modern-search#2019] [Q&A] [answered]
Is there a way to filter out results where the managed property FileType returns null?
<img width="132" alt="Screen Shot 2022-04-15 at 12 15 41 PM" src="https://user-images.githubusercontent.com/22969971/163606308-f44d529e-95a0-405a-b8cf-004282e6eb2a.png">

**Comment by wobba** [comment, marked as answer]
No. One option is to do an inverse if you know all the possible values. That said, what item are you seeing with no FileType associated to it?

`FileType<>doc* AND FileType<>xls* AND FileType<>ppt* AND FileType<>tif*` etc.

**Reply by JonoSuave** [reply]
I like that option. Document folders appears to be an example of where FileType is null

**Reply by wobba** [reply]
Folders can be matched with `ContentTypeId:0x0120*` which would also include OneNote's in SiteAssets and document sets I believe - as they are technically folders.

**Reply by StephenHoughton4** [reply]
From another Post, I was involved in Wobba provided an update on this.

This will show All Different FileTpes that have a value:
`FileType:*`

To invert it you could use prefix a - :
`-FileType:*`

may work, if not it works with other fields such as LastName etc.

**Comment by JonoSuave** [comment]
Thanks Mikael :)
